api/models/people.py

```diff
@@ -135,6 +138,43 @@ def __str__(self) -> str:
         return f"{self.personal_info.full_name}. Status: {getattr(self, 'status')}"
 
 
+# TODO the module is getting big. We can move Coordinator... etc into separate modules,
+#  which will also allow to rename all model modules into singular (because "people.py"
+#  will only contain `Person`).
+class CoordinatorQuerySet(models.QuerySet["Coordinator"]):
+    def annotate_with_count(self) -> "CoordinatorQuerySet":
```

Comment:
Let's rename it to `annotate_with_group_count`

Reply:
Done